### DISCUSSION TITLE
problem in Receive Address

### DISCUSSION THREAD

**Discussion by chiamila33** [Chia-Network/chia-blockchain#7267] [General]
hi 
I think have problem in Receive Address because when chose in pool or solo pool automatic change address and when I want chose new Receive Address and copy in Manage Your Farming Rewards Target Addresses place said this address used another people I can not change please help

**Comment by loppefaaret** [comment]
it is changing address, cause everytime you change the plot NFT (switching pool) you are spending a single mojo - but you are splitting coins from your wallet, and receiving change back on the previous receive address, prompting the wallet to generate a new address
this is normal behavior
now what exactly are you having problems with concerning your "Farming Rewards Target Address" ?
